--- a/drivers/net/wireless/ath/ath11k/dp_rx.c
+++ b/drivers/net/wireless/ath/ath11k/dp_rx.c
@@ -5835,7 +5835,8 @@ ath11k_dp_rx_mon_mpdu_pop(struct ath11k
 				  ent_desc->info1);
 		if (rxdma_err == HAL_REO_ENTR_RING_RXDMA_ECODE_FLUSH_REQUEST_ERR ||
 		    rxdma_err == HAL_REO_ENTR_RING_RXDMA_ECODE_MPDU_LEN_ERR ||
-		    rxdma_err == HAL_REO_ENTR_RING_RXDMA_ECODE_OVERFLOW_ERR) {
+		    rxdma_err == HAL_REO_ENTR_RING_RXDMA_ECODE_OVERFLOW_ERR ||
+		    rxdma_err == HAL_REO_ENTR_RING_RXDMA_ECODE_FCS_ERR) {
 			drop_mpdu = true;
 			pmon->rx_mon_stats.dest_mpdu_drop++;
 		}
